cumulus_library/studies/discovery/reference_sql/code_detection.sql

```diff
@@ -7,6 +7,90 @@
 -- ###########################################################
 
 CREATE TABLE discovery__code_sources AS
+SELECT *
+FROM (
+    VALUES (
+        'allergyintolerance',
+        'clinicalstatus',
+        '',
+        '',
+        ''
```

Comment:
This reference file has a lot of blanks - is there no coding data in the test data for allergies or these others? Procedure.reasonCode has something. But it's a rarity. Even `encounter.hospitalization.dischargedisposition` lost data.

Reply:
This is probably due to running against my test db - my internet access has been so bad from home over the last few days on VPN that i got bizaare timeouts trying to run this against cerner. I'll try it again tomorrow when i'm onsite.

Reply:
I think in CONTRIBUTING.md, we have a one-liner to run against the test data - does that generate good-enough results?

Reply:
updated w/ local db - still pretty blank-ful.